Gems/ROS2/Code/Source/Manipulation/JointsManipulationComponent.cpp

```diff
@@ -118,10 +119,12 @@ namespace ROS2
                 const AZStd::string jointName(frameComponent->GetJointName().GetCStr());
 
                 auto* hingeComponent = Utils::GetGameOrEditorComponent<PhysX::HingeJointComponent>(entity);
+                auto* prismaticComponent = Utils::GetGameOrEditorComponent<PhysX::PrismaticJointComponent>(entity);
```

Comment:
Please use 
```
azrtti_cast<PhysX::JointComponent*>(Utils::GetGameOrEditorComponent<PhysX::HingeJointComponent>(entity));
```
Instead of keeping multiple pointers type.